src/android/toga_android/widgets/box.py

```diff
@@ -1,6 +1,12 @@
 from .base import Widget
+from ..libs.activity import MainActivity
+from ..libs.android_widgets import RelativeLayout
 
 
 class Box(Widget):
     def create(self):
-        pass
+        self.native = RelativeLayout(MainActivity.singletonThis)
```

Comment:
No problems approving this as is; but I'd be interested in hearing your thoughts on the use of RelativeLayout vs using ViewGroup directly.

Reply:
Honestly, I'm only using `RelativeLayout` here because it's what I happened to use in earlier Python/Java Android demos.

I'm bullish on leaning on ViewGroup directly, and I'll see if I can take this PR in that direction.

Reply:
OK, I've taken a fresh look at the ViewGroup vs. RelativeLayout question. I'd prefer to merge this PR as-is, with RelativeLayout.

The reason for that is I want to work on the `ViewGroup` layout stuff next (i.e., now), and it's helpful for me to get more stuff merged, even if there's minor back-and-forth in the PRs.

The `ViewGroup` work is surely going to have its own feedback back-and-forth, and it'll be pretty substantial, and I'm itching to have `Button` work before that lands.

Reply:
Totally understood. And, as I said - no objection to landing this as is; I was just wondering if there was some benefit to RelativeLayout specifically.